pages/docs/v3/core-apis/saving-calls.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Persisting Plugin Calls
+description: How to save plugin calls in Capacitor
+---
+
+# Saving Plugin Calls
+
+In most cases, a plugin method will get invoked to perform a task and can finish immediately. But more complicated situations can require additional consideration.
+
+## Overview
+
+There are two reasons you might need a plugin call (`CAPPluginCall` on iOS or `PluginCall` on Android) to persist outside of the method in your plugin:
+
+1. To perform an asynchronous action, such as requesting permission from the system to access a protected API.
+2. To provide repeated updates back to the web environment, such as streaming live geolocation data.
+
+These two reasons overlap but they are not identical. Specifically, the difference is whether or not a call will need to complete (`resolve()` or `reject()`) more than once. The Capacitor bridge has to record each call that is made from JavaScript to native code so that it can match the result to the correct pending promise when the plugin returns it, and the default behavior is to erase this bookkeeping after a call completes once.
```

Comment:
```suggestion
There is a difference in handling saved calls depending on if you will need to complete (`resolve()` or `reject()`) more than once. The Capacitor bridge records each call that is made from JavaScript to native code so that it can match the result to the correct pending promise when the plugin returns it, and the default behavior is to erase this bookkeeping after a call completes once.
```